src/test/java/com/utils/CreatePropertiesFileTest.java

```diff
@@ -0,0 +1,51 @@
+package com.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CreatePropertiesFileTest {
+
+    private final String FILENAME = "application.properties";
+    private String propertySource;
+
+    private CreatePropertiesFile classUnderTest;
+    private Properties properties;
+
+    @BeforeEach
+    void setUp() {
+        this.classUnderTest = new CreatePropertiesFile();
+        properties = new Properties();
+        propertySource = "C:/Петко/udemy/java master class/Martin_Project/testResource/test.txt";
+    }
+
+    @Test
+    void whenLoadingProperties_thenExpectAllPropertiesToBeFilled() {
+        classUnderTest.loadPropertiesFromFile(FILENAME);
+    }
+
+    @Test
+    void givenRealProperties_whenLoading_thenNotNullExpectToBeReturned() {
+        assertNotNull(properties);
+    }
+
+    @Test
+    void givenRealName_whenGettingProperty_thenRightPropertyFileToBeExecuted() {
+        assertEquals(classUnderTest.getPropertyName(), FILENAME);
+    }
+
+    @Test
+    void givenPropertyKey_whenGettingProperty_thenRightPropertySourceToBeExecuted() {
+        assertEquals(classUnderTest.getProperty("testFilePath"), propertySource);
+    }
+
+    @Test
+    void whenReadingNull_thenExpectToThrowNullPointerException(){
```

Comment:
Should you expect this? Nullpointer is a runtime exception. You should know when your code can throw it and try to avoid it with some checks(check if something is null for example)